<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadFile</class>
 <widget class="QWidget" name="DownloadFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>603</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTreeWidget" name="tWSelectHomeWork">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>301</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="gBSelectStudent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Specific Student</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>33</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Student</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cBSelectStudent">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QTreeWidget" name="tWSelectClass">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>311</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>20</width>
     <height>541</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Class</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBClose">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>550</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Home Work</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBDownload">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>530</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Download Home Work</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pBClose</sender>
   <signal>clicked()</signal>
   <receiver>DownloadFile</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>884</x>
     <y>573</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
